app/script/calling/entities/EFlow.js

```diff
@@ -0,0 +1,1361 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.calling = z.calling || {};
+window.z.calling.entities = z.calling.entities || {};
+
+const E_FLOW_CONFIG = {
+  DATA_CHANNEL_LABEL: 'calling-3.0',
```

Comment:
these const leak into the global space. so either add a closure or make them static